resources/views/layout/master.blade.php

```diff
@@ -0,0 +1,30 @@
+   </style>
+</head>
+<body>
+    <div class="relative sm:flex sm:justify-center sm:items-center min-h-screen bg-dots-darker bg-center bg-gray-100 dark:bg-dots-lighter dark:bg-gray-900 selection:bg-red-500 selection:text-white">
+    
+        <div class="container">
+            @yield('Addpost')
+        </div>
+
+        <div class="container">
+            @yield('Viewpost')
+        </div>
+
+        <div class="container">
+            @yield('Editpost')
```

Comment:
Please only use one yield directive.
This can be reused.
We usually name it 'content'
@yield('content')

Then on your child blades just call it as
@section('content')
@endsection